test/HostActivationTests/GivenThatICareAboutPortableAppActivation.cs

```diff
@@ -74,7 +74,7 @@ public void Muxer_Exec_activation_of_Build_Output_Portable_DLL_with_DepsJson_Loc
             dotnet.Exec("exec", "--runtimeconfig", runtimeConfig, appDll)
                 .CaptureStdErr()
                 .CaptureStdOut()
-                .Execute()
+                .Execute(true)
```

Comment:
Can you point me to ShouldFail in CLI? I don't see any results at https://github.com/dotnet/cli/search?utf8=%E2%9C%93&q=ShouldFail.